### PR TITLE
Develope

### PR DESCRIPTION
**Pull request by guest2acute**
Please check my code

### CODE REVIEW

File: src/pages_crm/pages/index.tsx [hunk 1]
```diff
@@ -10,6 +10,10 @@ import { BecomePartnerFormWrapper } from "./becomePartner";
 import OTPVerificationPage from "./otpVerificationDialog";
 import AadharVerification from "./aadharVerification";
 import Equifax from "./equifax";
+import { InquiryViewFormWrapper } from "./inquiryFormActions/inquiryFormView";
```
Comment (devarsh) [inline src/pages_crm/pages/index.tsx:13]: Move edit forms and view forms from CRM to LOS

File: src/components/dataTable/tableHeaderToolbar.tsx [hunk 2]
```diff
@@ -1,12 +1,15 @@
+import { useState } from "react";
```
Comment (devarsh) [inline src/components/dataTable/tableHeaderToolbar.tsx:1]: Remove your changes and only keep original changes

File: src/pages_los/pages/tabView/tabPanelContent/detailsTabs/tabs.tsx [hunk 3]
```diff
@@ -7,6 +7,9 @@ import { LeadInformationTab } from "./tabInformation";
 import { LeadCustomerTab } from "./tabCustomer";
 import { useStyles } from "./style";
 
+import { InquiryViewFormWrapper } from "pages_crm/pages/inquiryFormActions/inquiryFormView";
```
Comment (devarsh) [inline src/pages_los/pages/tabView/tabPanelContent/detailsTabs/tabs.tsx:10]: Change navigation path from CRM to LOS

File: src/pages_crm/pages/index.tsx (hunk 1 above)
Comment (guest2acute) [reply src/pages_crm/pages/index.tsx:13]: form moved successfully

File: src/components/dataTable/tableHeaderToolbar.tsx (hunk 2 above)
Comment (guest2acute) [reply src/components/dataTable/tableHeaderToolbar.tsx:1]: changes replace with original change

File: src/pages_los/pages/tabView/tabPanelContent/detailsTabs/tabs.tsx (hunk 3 above)
Comment (guest2acute) [reply src/pages_los/pages/tabView/tabPanelContent/detailsTabs/tabs.tsx:10]: navigation changes done

File: src/pages_los/pages/inquiryFormActions/inquiryFormEdit.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+    Promise.all([
+      APISDK.getInquiryFormDataForEdit(),
+      APISDK.getInquiryFormMetaDataForEditOnly(),
+    ])
+      .then(function (responses) {
+        Promise.all(responses).then((data) => {
+          if (data[0].status === "success") {
+            setLoading(false);
+            setMetaData(data[1]);
```
Comment (devarsh) [inline src/pages_los/pages/inquiryFormActions/inquiryFormEdit.tsx:30]: Do error checking for both data [0] & data [1]

File: src/pages_los/pages/inquiryFormActions/inquiryFormEdit.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+            setFormEditableValues(data[0].data);
+          } else {
+            setLoading(false);
+            setError(data[0].data.error_msg);
+          }
+        });
+      })
+      .catch(function (error) {
+        setLoading(false);
+        console.log(error);
```
Comment (devarsh) [inline src/pages_los/pages/inquiryFormActions/inquiryFormEdit.tsx:40]: Remove this console.log

File: src/pages_los/pages/inquiryFormActions/inquiryFormView.tsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+  useEffect(() => {
+    setLoading(true);
+    Promise.all([
+      APISDK.getInquiryFormDataForView(),
+      APISDK.getInquiryFormMetaDataForViewOnly(),
+    ])
+      .then(function (responses) {
+        Promise.all(responses).then((data) => {
+          if (data[0].status === "success") {
```
Comment (devarsh) [inline src/pages_los/pages/inquiryFormActions/inquiryFormView.tsx:28]: Error checking for data[0] & data[1] success status.

File: src/pages_los/pages/questionnaireFormActions/questionnaireFormEdit.tsx [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  useEffect(() => {
+    setLoading(true);
+
+    Promise.all([
+      APISDK.getQuestionnairesFormDataForEdit(),
+      APISDK.getQuestannaiareFormMetaDataForEditOnly(),
+    ])
+      .then(function (responses) {
+        Promise.all(responses).then((data) => {
+          if (data[0].status === "success") {
```
Comment (devarsh) [inline src/pages_los/pages/questionnaireFormActions/questionnaireFormEdit.tsx:28]: Error checking for data[0] & data[1]

File: src/pages_los/pages/questionnaireFormActions/questionnaireFormView.tsx [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+  useEffect(() => {
+    setLoading(true);
+    Promise.all([
+      APISDK.getQuestionnairesFormDataForView(),
+      APISDK.getQuestionnaireFormMetaDataForViewOnly(),
+    ])
+      .then(function (responses) {
+        Promise.all(responses).then((data) => {
+          if (data[0].status === "success") {
```
Comment (devarsh) [inline src/pages_los/pages/questionnaireFormActions/questionnaireFormView.tsx:24]: Error checking of data[0] & data[1]

**Review by devarsh — COMMENTED** [review]
Pl perform necessary error checking and resubmit.

**Comment by guest2acute** [comment]
Please check the changes

**Pull request merged**